starter_code/css/mobile.css

```diff
@@ -0,0 +1,240 @@
+@media(min-width: 320px){
+
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+  max-width: 100%;
+}
+.logo {
+  font-size: 32px;
+  font-weight: 200;
+  text-align: center;
+}
+
+header {
+  padding: 5px 0;
+  overflow: hidden;
+  text-align: center;
+}
+
+header .logo {
+  text-align: center; 
+}
+
+header nav {
+  margin-top: 15px;
+  text-align: center;
+  font-size: 24px;
+}
+
+header nav ul li {
+  display: block;
+  text-align: center;
+  font-size: 24px;
+}
+
+
+#hero {
+  height: 400px;
+  padding-top: 32px;
+}
+
+
+#hero h2 {
+  font-size: 40px;
+  text-align: center;
+  margin: 20px 0 40px;
+  font-weight: 600;
+}
+.button {
+  border-radius: 5px;
+  font-weight: 600;
+  padding: 12px 0;
+  width: 200px;
+  font-size: 22px;
+  display: block;
+  margin: 0 auto;
+}
+
+.blue {
+  font-size: 22px;
+  font-weight: 600px;
+}
+
+button .blue {
+  width: 200px;
+}
+#benefits {
+  padding: 20px;
+  text-align: center;
+  width: 100%;
+}
+h3 {
+  font-size: 40px;
+  font-weight: 600;
+  margin-bottom: 40px;
+}
+
+#benefits p, #benefits ul {
+  font-size: 24px;
+  padding: 10px;
+}
+
+#benefits ul {
+  list-style: disc;
+  margin: 10px;
+}
+#testimonial {
+  background: url(../images/testimonial_bg.jpg) no-repeat top center #071324;
+  padding: 20px;
+  
+}
+blockquote {
+  font-size: 24px;
+  font-weight: 300;
+  font-style: italic;
+  text-align: center;
+  width: 100%;
+  margin: 0 auto;
+  padding: 10px;
+}
+blockquote span {
+  font-weight: 600;
+  display: block;
+  font-style: normal;
+}
+#main {
+  margin-top: 80px;
+}
+#posts {
+  width: 100%;
+  display: block;
+  margin-right: 10px;
+  vertical-align: top;
+}
+#sidebar {
+  width: 100%;
+  display: block;
+  vertical-align: top;
+}
+.post {
+  border-bottom: 1px solid #043049;
+  padding-bottom: 20px;
+  margin: 20px;
+}
+.post h2 {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 1.3;
+  margin-bottom: 20px;
+  text-align: center;
+}
+.post h2 a {
+  color: #043049;
+}
+
+.post img {
+  margin-bottom: 20px;
+  width: 100%;
+}
+.post p {
+  font-size: 22px;
+  line-height: 1.333;
+}
+.post p:first-of-type:first-letter {
+  font-size: 40px;
+  padding-top: 4px;
+  padding-right: 8px;
+  padding-left: 3px;
+  font-weight: 600;
+}
+.readmore {
+  text-align: right;
+  margin-top: 30px;
+}
+.readmore a {
+  color: #043049;
+  font-weight: 600;
+}
+
+/*.button {
```

Comment:
Its best to remove any commented out styles like this unless you intend to add them back.